bookwyrm/views/feed.py

```diff
@@ -230,3 +248,16 @@ def get_suggested_books(user, max_books=5):
         suggested_books.append(shelf_preview)
         book_count += len(shelf_preview["books"])
     return suggested_books
+
+
+def filter_stream_by_status_type(activities, allowed_types=[]):
+    if "review" not in allowed_types:
+        activities = activities.filter(Q(review__isnull=True))
```

Comment:
I don't know if it's the best way to filter activities, I don't think I have access to the class name of each object to filter out.

Reply:
I can't think of a better way to do this

Reply:
I just realized boosts are getting filtered as an "everything else" type, rather than under the status type of the boosted status. You can access the type boosted status with, for example: `boost__boosted_status__review__isnull=True`